packages/@okta/vuepress-site/docs/guides/configure-native-sso/main/index.md

```diff
@@ -0,0 +1,439 @@
+
+If you need help or have an issue, post a question on the [Okta Developer Forum](https://devforum.okta.com).
+
+# Before you begin
+
+This guide assumes that you:
+
+* Have an Okta Developer Edition organiztion. Don't have one? [Create one for free](https://developer.okta.com/signup).
+* Have the Native SSO feature enabled for your org. From the left navigation pane in the Admin Console, go to **Settings** > **Features**, locate the Native SSO slider, and slide to enable.
+
+## Native SSO flow
+
+![Native SSO flow](/img/native_SSO_flow.png)
+
+<!-- Source for image. Generated using http://www.plantuml.com/plantuml/uml/
+
+skinparam monochrome true
+
+participant "Native App #1" as app1
+participant "Native App #2" as app2
+participant "Browser" as browser
+participant "Authorization Server (Okta)" as okta
+
+app1 -> browser: Start authorization flow
+browser -> okta: 1. Send request to /authorize with `device_sso` scope
+okta -> browser: 2. Prompts the user to authenticate
+browser -> okta: 3. Enter credentials to authorize and provide consent
+okta -> app1: 4. Returns the authorization code
+app1 -> okta: 5. Send the authorization code in the request for tokens
+okta -> app1: 6. Returns `id_token`, `refresh_token`, `access_token`, and `device_secret`
+app2 -> okta: 7. Request for an `access_token` and `refresh_token`
+okta -> app2: 8. Returns an `access_token` and `refresh_token`
+
+-->
+
+1. Native app 1 starts by redirecting the user's browser to the [authorization server](/docs/concepts/auth-servers/) `/authorize` endpoint and requests the `device_sso` scope.
+1. The user is prompted to authenticate.
+1. The user enters their credentials to allow the application to access resources protected by scopes.
+1. The authorization server returns the authorization code for Native app 1.
+1. The client exchanges the authorization code for tokens.
+1. The authorization server returns the tokens (`id_token`, `refresh_token`, and `access_token`) and the `device_secret` in the response.
+1. Native app 2 makes a request for a `refresh_token` and `access_token`. The request contains the `id_token` and the `device_secret`.
+
+	Native app 2, and any other client that participates in the Native SSO flow, can use the `id_token` and the `device_secret` obtained from the initial client that authenticated (see the following diagram). To sign in automatically, the clients can use the `id_token` and `device_secret` and exchange them for tokens by making a `/token` request.
+
+	![ID token and device secret use](/img/nativeSSO_flow2.png)
+
+8. The authorization server returns a new set of refresh and access tokens specifically for Native app 2. This key part in the Native SSO flow enables a user to be automatically signed in without requiring any user action.
+
+To use the Native SSO functionality, you need to:
+
+* Configure Native SSO for your Okta org
+* Configure the Token Exchange grant type for all clients participating in the Native SSO flow
```

Comment:
this doesn't match the title below. they should be the same. so either use this one down below, or use this one up here in the list: ## Configure your client to participate in Native SSO

Reply:
Fixed